<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sap</class>
 <widget class="QMainWindow" name="sap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>457</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Simple Accounting Program</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabMain">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>571</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tabDatabase">
     <attribute name="title">
      <string>Database</string>
     </attribute>
     <widget class="QGroupBox" name="grpDatabase">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>311</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>Manage Databases</string>
      </property>
      <widget class="QLineEdit" name="txtDbNew">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>30</y>
         <width>113</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btnDbNew">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>30</y>
         <width>61</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cmbDbDelete">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>70</y>
         <width>111</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btnDbDelete">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>70</y>
         <width>61</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Create Data</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Delete Data</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="grpExportImport">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>190</y>
        <width>311</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Export/Import Data</string>
      </property>
      <widget class="QComboBox" name="cmbDbExport">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>111</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Data to Export</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnDbExport">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>20</y>
         <width>61</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnDbImport">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>70</y>
         <width>61</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Import</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cmbDbImport">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>70</y>
         <width>111</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Import to Data</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="grpDbSelect">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Select Data </string>
      </property>
      <widget class="QPushButton" name="btnDbExisting">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>61</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Use</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cmbDbExisting">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>131</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tabInput">
     <attribute name="title">
      <string>Input</string>
     </attribute>
     <widget class="QGroupBox" name="grpInput">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>541</width>
        <height>231</height>
       </rect>
      </property>
      <property name="title">
       <string>Input Transaksi</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Deksripsi Transaksi</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtTrsDeskrip">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>401</width>
         <height>23</height>
        </rect>
       </property>
       <property name="maxLength">
        <number>32</number>
       </property>
      </widget>
      <widget class="QComboBox" name="cmbTrsDebet">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>90</y>
         <width>401</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>90</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Debit</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Nilai Transaksi</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtTrsNilai">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>60</y>
         <width>401</width>
         <height>23</height>
        </rect>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly|Qt::ImhFormattedNumbersOnly|Qt::ImhPreferNumbers</set>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>32</number>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateTrsTanggal">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>160</y>
         <width>110</width>
         <height>23</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="displayFormat">
        <string>yyyy-M-d</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="showGroupSeparator" stdset="0">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Tanggal Transaksi</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnTrsSave">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>200</y>
         <width>85</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnTrsClear">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>200</y>
         <width>85</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnTrsNow">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>160</y>
         <width>85</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Now</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>120</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Kredit</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cmbTrsKredit">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>120</y>
         <width>401</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tabSearch">
     <attribute name="title">
      <string>Search</string>
     </attribute>
     <widget class="QGroupBox" name="grpCari">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>541</width>
        <height>291</height>
       </rect>
      </property>
      <property name="title">
       <string>Cari Data</string>
      </property>
      <widget class="QLineEdit" name="txtCariDeskrip">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>371</width>
         <height>23</height>
        </rect>
       </property>
       <property name="maxLength">
        <number>32</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtCariNilai">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>60</y>
         <width>371</width>
         <height>23</height>
        </rect>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly|Qt::ImhFormattedNumbersOnly|Qt::ImhPreferNumbers</set>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>32</number>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateCariTanggal">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>150</y>
         <width>110</width>
         <height>23</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="displayFormat">
        <string>yyyy-M-d</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="showGroupSeparator" stdset="0">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="btnCari">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>190</y>
         <width>85</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnCariClear">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>190</y>
         <width>85</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnCariNow">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>150</y>
         <width>85</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Now</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rbtCariDeskrip">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>97</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Deskripsi</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="rbtCariNilai">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>97</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Nilai</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rbtCariTanggal">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>97</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Tanggal</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cmbCariDebet">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>90</y>
         <width>371</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QRadioButton" name="rbtCariDebet">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>97</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Debet</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cmbCariKredit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>120</y>
         <width>371</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QRadioButton" name="rbtCariKredit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>97</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Kredit</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tabEdit">
     <attribute name="title">
      <string>Edit</string>
     </attribute>
     <widget class="QGroupBox" name="grpEdit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>541</width>
        <height>271</height>
       </rect>
      </property>
      <property name="title">
       <string>Edit Transaksi</string>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Deksripsi Transaksi</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtEditDeskrip">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>70</y>
         <width>401</width>
         <height>23</height>
        </rect>
       </property>
       <property name="maxLength">
        <number>32</number>
       </property>
      </widget>
      <widget class="QComboBox" name="cmbEditDebet">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>130</y>
         <width>401</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>130</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Debit</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Nilai Transaksi</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtEditNilai">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>100</y>
         <width>401</width>
         <height>23</height>
        </rect>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly|Qt::ImhFormattedNumbersOnly|Qt::ImhPreferNumbers</set>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>32</number>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEditTanggal">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>200</y>
         <width>110</width>
         <height>23</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="displayFormat">
        <string>yyyy-M-d</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="showGroupSeparator" stdset="0">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Tanggal Transaksi</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnEditChange">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>240</y>
         <width>85</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnEditNow">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>200</y>
         <width>85</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Now</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Data ID</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtEditID">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>20</y>
         <width>181</width>
         <height>23</height>
        </rect>
       </property>
       <property name="maxLength">
        <number>32</number>
       </property>
      </widget>
      <widget class="QPushButton" name="btnEditShow">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>20</y>
         <width>85</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Show</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnEditDelete">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>240</y>
         <width>85</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>160</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Kredit</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cmbEditKredit">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>160</y>
         <width>401</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>586</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionTableJurnal"/>
    <addaction name="actionTableAkun"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionTableJurnal">
   <property name="text">
    <string>Table Jurnal</string>
   </property>
  </action>
  <action name="actionTableAkun">
   <property name="text">
    <string>Table Akun</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
